app/models/document.rb

```diff
@@ -38,10 +37,6 @@ def truncated_description
     description.gsub(/\.\s[A-Z].*/, '.') if description.present?
   end
 
-  def summary
```

Comment:
Was this only used by atom feeds? 😮

Reply:
I searched the codebase for 'summary' and Document#summary seems to only be used in EntryPresenter#summary.

A very similar method lives in SearchResultPresenter#summary_text which uses Document#truncated_description directly.

It's not great that there are two methods in different places doing similar things but this is just one small step in many refactoring to hopefully make things better